AEPTarget/Sources/TargetDeliveryRequestBuilder.swift

```diff
@@ -54,7 +55,17 @@ enum TargetDeliveryRequestBuilder {
             property = Property(token: propertyToken)
         }
 
-        return TargetDeliveryRequest(id: targetIDs, context: context, experienceCloud: experienceCloud, prefetch: prefetch, execute: execute, notifications: notifications, environmentId: environmentId, property: property)
+        var qaMode: [String: AnyCodable]?
+        if
+            let qaModeParameters = qaModeParameters,
+           !qaModeParameters.isEmpty,
+           let qaModeData = qaModeParameters.data(using: .utf8)
+        {
+            let qaModeDict = try? JSONSerialization.jsonObject(with: qaModeData, options: []) as? [String: Any]
+            qaMode = AnyCodable.from(dictionary: qaModeDict?[TargetConstants.TargetJson.QA_MODE] as? [String: Any])
+        }
```

Comment:
this section is begging to be shared internally in an extension (json/dictionary/string conversions).  maybe something we can bring up with the core team to get into services?

Reply:
Sure, let's discuss this further in the SDK guild meeting.